styles.css

```diff
@@ -0,0 +1,213 @@
+.container {
+  width: 100vw;
+  height: 100vh;
+  font-family: 'Poppins', sans-serif;
+  background-color: lightgreen;
+
+
+
+}
+
+
+.first_block ul {
+  display: flex;
+  list-style: none;
+  text-decoration: none;
+
+}
+
+.list > .startly{
+  display: flex;
+  flex: 2;
+}
+
+
+.list > .try{
+  display: flex;
+  flex: 1;
+}
+
+
+.first_block {
+  min-width: 100px;
+  min-height: 70px;
+  font-size: 0.5em;
+  font-weight:bold;
+}
+
+
+
+.second_block {
+  min-width: 100px;
+  min-height: 200px;
+  border-bottom: solid 1px gray;
+  text-align: center;
+}
+
+.second_block  {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+
+
+}
+
+
+.second_block sub {
+  font-weight: bold;
+}
+
+
+.second_block a {
+  border-radius: 50px;
+  border:1px solid grey;
+
+}
+
+
+
+.third_block {
+  border-bottom: solid 1px gray;
+  font-style: italic;
+
+}
+
+.third_block blockquote {
+  display: flex;
+  justify-content: center;
+}
+
+
+.third_block blockquote:before, .third_block blockquote:after {
```

Comment:
Good use of before, I wonder if this could have been selected with just blockquote rather than .third_block blockquote.